core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/negotiation/ConsumerContractNegotiationManagerImpl.java

```diff
@@ -248,6 +242,15 @@ public NegotiationResult declined(ClaimToken token, String negotiationId) {
         return NegotiationResult.success(negotiation);
     }
 
+    private ContractNegotiation findContractNegotiationById(String negotiationId) {
```

Comment:
Should this method used also on `declined` method on provider side?

Reply:
Yes, you are right, I will add it. Thanks!